hatchet_sdk/context/context.py

```diff
@@ -54,7 +63,7 @@ def _prepare_workflow_options(
         if options is not None and "additional_metadata" in options:
             meta = options["additional_metadata"]
 
-        trigger_options: TriggerWorkflowOptions = {
+        trigger_options: TriggerWorkflowOptions = {  # type: ignore[typeddict-item]
```

Comment:
this is here because `namespace` isn't set - if we do set it, it causes problems downstream. I think Pydantic here would be helpful instead of using typed dicts